### ISSUE TITLE
Do I need to incorporate LVIS novel categories (i.e., rare categories) when training the detector section （detpro）on LVIS?

### ISSUE THREAD

**Issue by LMJ231** [SysCV/ovtrack#9]
I noticed in your paper that you mentioned, 'In the validation set of TAO, 295 of the overlapping classes are present. 35 of these classes are defined as rare, which serve as our Cnovel.' and 'In the test set of TAO, there are 324 of the overlapping classes mentioned above present. 33 of these classes are defined as rare, which serve as our Cnovel.' I'd like to inquire whether the novel classes set in your benchmark (TAO's novel classes) overlap with LVIS's novel classes?

**Comment by siyliepfl** [comment]
Sorry for the late reply. Yes, they are overlapping in the sense that the TAO novel classes we used are the subset of LVIS classes.
